installer/scripts/tailfilereader.rb

```diff
@@ -0,0 +1,396 @@
+require 'optparse'
+require 'logger'
+
+module Tailscript
+
+  class NewTail  
+    def initialize(paths)
+      @paths = paths
+      @tails = {}
+      @pos_file = $options[:pos_file] 
+      @read_from_head = $options[:read_from_head]
+      @pf = nil
+      @pf_file = nil
+
+      $log = Logger.new(STDERR)
```

Comment:
I don't think we want to overwrite the global $log variable provided by the engine.